src/app/api/_api-services/onchain_db_service/subsquid_service/index.ts

```
@@ -940,6 +948,45 @@ export class SubsquidService extends SubsquidUtils {
+			console.error(`Error fetching network governance analytics stats from Subsquid: ${subsquidErr}`);
+			throw new APIError(ERROR_CODES.INTERNAL_SERVER_ERROR, StatusCodes.INTERNAL_SERVER_ERROR, 'Error fetching network governance analytics stats from Subsquid');
+		}
+
+		return {
+			totalProposals: subsquidData.totalProposals.totalCount,
+			approvedProposals: subsquidData.approvedProposals.totalCount
+		};
+	}
+
+	static async GetGovAnalyticsReferendumOutcome({ network, trackNo }: { network: ENetwork; trackNo?: number }): Promise<IGovAnalyticsReferendumOutcome> {
+		const gqlClient = this.subsquidGqlClient(network);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)